tracing/otel/provider.go

```diff
@@ -251,3 +264,38 @@ func (o *OtelProvider) tracerFromContext(ctx context.Context) trace.Tracer {
 type getTraceLogger interface {
 	LogFromCtx(ctx context.Context) *zerolog.Logger
 }
+
+// Despite OTEL SDK's documentation, it doesn't seem to want to use the standard
+// OTEL env vars when deployed into GKE. Instead of trying to debug/find
+// the obscure piece of documentation on why this happens, we've instead
+// decided to handle some of the env vars ourself.
+func samplerFromEnv() (sdktrace.Sampler, error) {
+	sampler, _ := os.LookupEnv(otelSamplerEnvVar)
+	samplerArg, _ := os.LookupEnv(otelSamplerArgEnvVar)
```

Comment:
If you don't need to distinguish between an empty value and an unset value than you can use `os.Getenv`

Reply:
:D Knew there was one like that, but intelisense failed to show it (and I was too lazy to google).